StackExchange.Redis/StackExchange/Redis/ConfigurationOptions.cs

```diff
@@ -65,7 +66,18 @@ internal static Proxy ParseProxy(string key, string value)
                 if (!Enum.TryParse(value, true, out tmp)) throw new ArgumentOutOfRangeException("Keyword '" + key + "' requires a proxy value");
                 return tmp;
             }
+#if !CORE_CLR
+            internal static SslProtocols ParseSslProtocols(string key, string value)
+            {
+                SslProtocols tmp;
+                //Flags expect commas as separators, but we need to use '|' since commas are already used in the connection string to mean something else
+                value = value?.Replace("|", ","); 
+
+                if (!Enum.TryParse(value, true, out tmp)) throw new ArgumentOutOfRangeException("Keyword '" + key + "' requires an SslProtocol value (multiple values separated by '|').");
 
+                return tmp;                
+            }
+#endif

```

Comment:
The CORE_CLR was purely because I don't have the right tools locally to test my changes for core clr can thus couldn't even compile to see if I got everything correct.  I suspect Core CLR would support the same concept but I don't have a way to validate currently.